### PR TITLE
Add `mithril-client-wasm` tests in multi platform test

### PR DESCRIPTION
**Pull request by dlachaumepalo**
## Content
This PR includes tests implementation for `mithril-client-wasm`. This tests are executed in headless mode with Google Chrome and Firefox from the manual workflow `Mithril Client multi-platform test`.

Two scripts were added:
- `run-wasm-tests-browser-headless.py`: call by the manual workflow to run browsers in headless mode and save the output to a html file with the test results. It takes the browser as argument (`chrome` or `firefox`).
- `parse-wasm-headless-tests-results.sh`: call by the manual workflow to handle both browsers test results

## Pre-submit checklist

- Branch
  - [x] Tests are provided (if possible)
  - [x] Crates versions are updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [x] Update README file (if relevant)
  - [ ] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

## Issue(s)
Close to #1408

### CODE REVIEW

File: mithril-client-wasm/www-test/package.json
Comment (Alenar) [inline mithril-client-wasm/www-test/package.json:1]: You should update the name, description, repository, keywords, ... with our information.

File: mithril-client-wasm/www-test/index.js
Comment (Alenar) [inline mithril-client-wasm/www-test/index.js:1]: Can you use `;` for the end of lines as we do for other js scripts usually ?

File: mithril-client-wasm/www-test/index.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+await initMithrilClient()
+const aggregator_endpoint = process.env.AGGREGATOR_ENDPOINT
+const genesis_verification_key = process.env.GENESIS_VERIFICATION_KEY
+let client
+let test_name
+let test_number = 1
+
+// Test 'MithrilClient constructor'
+try {
```
Comment (Alenar) [inline mithril-client-wasm/www-test/index.js:38]: You can factor those try/catch using a function, something like this should work:

```js
function test(name, step,  fun) {
  try {
    fun();
    display_test_result_in_dom(step, name, "OK");
  } catch (error) {
    handle_error(test_number, test_name, error);
  }
}

let client;
let step = 1;

test("constructor", step, () => {
  client = new MithrilClient(aggregator_endpoint, genesis_verification_key);
});

step++;
test("list_snapshots", step, () => {
  snapshots = await client.list_snapshots();
  console.log("snapshots", snapshots);
});
```

File: mithril-client-wasm/www-test/.bin/create-wasm-app.js
Comment (Alenar) [inline mithril-client-wasm/www-test/.bin/create-wasm-app.js:1]: What is the purpose of this script ? Can't we just remove it ? _(and from the package.json)_

File: mithril-client-wasm/www-test/index.js (hunk 1 above)
Comment (dlachaumepalo) [reply mithril-client-wasm/www-test/index.js]: Thanks 🙏

**Review by Alenar — APPROVED** [review]

**Pull request merged**